opentracing-api/src/main/java/io/opentracing/LogField.java

```diff
@@ -0,0 +1,20 @@
+package io.opentracing;
+
+/**
+ * LogField represents a single key:value pair in a Span.log() record.
+ *
+ * The key must always be a String. All Tracer implementations must support values bool, numeric, and String values;
+ * some may also support arbitrary Object values.
+ */
+public class LogField<V> {
+    private final String key;
+    private final V value;
+
+    public LogField(String key, V value) {
```

Comment:
Should this throw if key is null or is checking this up to the trace implementation?